src/bridge/cockpitpackages.c

```diff
@@ -918,7 +918,7 @@ package_content (CockpitPackages *packages,
               cockpit_web_response_error (response, 404, NULL, NULL);
               goto out;
             }
-          else if (package->unavailable)
+          else if (package && package->unavailable)
```

Comment:
again, it's already checked later: https://github.com/cockpit-project/cockpit/pull/6024/files#diff-13e60e206825248c0748834f531b7f1bR958
There is no catch-all clause here to generate errors, but I would prefer to make sure we don't read a NULL pointer.